crates/stdarch-gen/neon.spec

```diff
@@ -652,6 +652,126 @@ validate -1.0, 2.0, -3.0, 4.0
 aarch64 = fcvtxn
 generate float32x2_t:float64x2_t:float32x4_t
 
+/// Floating-point convert to signed fixed-point, rounding toward zero
+name = vcvt
+double-suffixes
+fn = simd_cast
+a = -1.0, 2.0, -3.0, 4.0
```

Comment:
Wouldn't it make more sense to use .5 and co. to actually test the rounding (for all of these instructions)?

Reply:
Yes, but I don’t have a good way to test the results. How can we simulate rounding towards to even?

Reply:
e.g.:

a = -1.5, 2.5, -3.5, 4.5
validate = -2, 2, -4, 4

I'm confused what the problem is?

Reply:
Sorry I forgot that this is rounding to an integer. I will correct accordingly